packages/react-components/react-positioning/etc/react-positioning.api.md

```diff
@@ -109,10 +107,8 @@ export type PositioningVirtualElement = {
 // @public (undocumented)
 export function resolvePositioningShorthand(shorthand: PositioningShorthand | undefined | null): Readonly<PositioningProps>;
 
-// Warning: (ae-internal-missing-underscore) The name "usePositioning" should be prefixed with an underscore because the declaration is marked as @internal
-//
-// @internal
-export function usePositioning(options?: UsePopperOptions): {
+// @public (undocumented)
+export function usePositioning(options: UsePositioningOptions): {
```

Comment:
It was not public before, are we making it public?

Reply:
good catch 👍